src/Language/Haskell/GhclibParserEx/DynFlags.hs

```diff
@@ -20,6 +22,34 @@ import StringBuffer
 import HscTypes
 import GHC.LanguageExtensions.Type
 import Data.List
+import Data.Maybe
+import qualified Data.Map as Map
+
+-- | Parse a GHC extension
+readExtension :: String -> Maybe Extension
+readExtension x = Map.lookup x exts
```

Comment:
```suggestion
readExtension = \x -> Map.lookup x exts
```

The difference is the difference between computing the `exts` table each and every time, or doing it once.

Reply:
Excellent.